src/less/page-grid/base/page-grid.less

```diff
@@ -0,0 +1,42 @@
+.page-grid-container {
+    display: flex;
+    justify-content: center;
+    margin: 0 @spacing-small;
+}
```

Comment:
Is this needed? Can we not put this in the page grid class? I think we can just do `justify-content: center` there and it should work I assume? https://css-tricks.com/snippets/css/complete-guide-grid/#justify-content

Reply:
It needs to be a flex item/child for that to work though, right? 

Try here: https://ebay.github.io/skin/?debug=1

Reply:
I think we can do something with margin auto, either way... I think it's fine as is.

Reply:
`margin: 0 auto` would center the grid element... but then we lose the ability to add the outer margins.

Reply:
Unless there's something clever we can do with min, max, minmax etc for the horizonatal value...

Reply:
I think we could try something like 
```css
.page-grid-container {
  display: grid;
  place-items: center;
}
```

I think that would center things correctly, wouldn't it?

Reply:
Yeah, the good thing about having a container is that it gives us this level of flexibility if we want it.

Reply:
I'm going to merge this. We can always refine as we go.